README.md

```diff
@@ -44,5 +44,5 @@ The output of this script should give you the instructions you will need.
 - (Possible) Add in rake task for adding build to CI
 - Add generators for CRUD
 - Use dropdown menu for user actions
-- Add in mailer defaults for dev (mailcatcher), staging, and production (jazz)
+- Add in mailer defaults for staging, and production (jazz)
```

Comment:
Put the corresponding change in the list of things the template does. EG: Provides letter opener for mail stuffs